.github/workflows/scan.yml

```
@@ -0,0 +1,24 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  push:
+  pull_request:
```

1. I would also add `schedule` so this runs regularly.
2. I would use `paths` for `push` and `pull_request` so this only runs when workflows are modified. This avoids unrelated PRs that don't modify workflows to receive CI failures for workflows if something unrelated changes.
```suggestion
  schedule:
    # Daily
    - cron: "23 7 * * *"
  push:
    paths:
      - .github/workflows/*
  pull_request:
    paths:
      - .github/workflows/*
```

---

@felixfontein Thanks, that's exactly the kind of feedback I was hoping for. I am going to go ahead and accept this change. If there are other suggestions we can move forward from there.